.openshift-ci/jobs/update-support-packages/env.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -x
+
+# to ensure that the locale is consistent between GCP VMs
+# we need to set this here. If there are issues with the
+# locale, there can be some additional logging which
+# breaks the integration tests.
+export LC_ALL=C
+
+# JOB_ID is used as a random suffix to distinguish VMs
+export JOB_ID="${PROW_JOB_ID:0:8}"
```

Comment:
Is BUILD_ID not enough, just to simplify substring part?

Reply:
It is. I now use BUILD_ID.

Reply:
Actually this didn't work. It made the VM name too long.

Reply:
Interesting. Just for me to know what is the name limit?

Reply:
I am not sure exactly. There was an error message specifying a regex that the VM name had to conform to. Part of the regex specified up to 61 repeats of [-a-z0-9]. I think the limit is a bit longer than that though.